src/utils/eventLevels.js

```diff
@@ -48,7 +48,8 @@ export function eventLevels(rowSegments, limit = Infinity) {
     if (j >= limit) {
       extra.push(seg);
     } else {
-      (levels[j] || (levels[j] = [])).push(seg);
+      let level = levels[j] || (levels[j] = []);
```

Comment:
no longer need the level ivar. replace with `(levels[j] || (levels[j] = [])).push({ ...seg, level: j });`